.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+# Bug Report
+
```

Comment:
Unrelated to code, but the PR itself. Could you rename the title from

```diff
Template: add template for issues and pull requests
chore(github): Add template for issues and pull requests
```

To follow the [conventional commits mentioned in the contributing file](https://github.com/remoteoss/json-schema-form/blob/main/CONTRIBUTING.md#merging-a-pr)

Reply:
Added a section on commit structure to the documentation, and included a clear example for reference.